umodbus/asynchronous/serial.py

```diff
@@ -0,0 +1,397 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2019, Pycom Limited.
+#
+# This software is licensed under the GNU GPL version 3 or any
+# later version, with permitted additional terms. For more information
+# see the Pycom Licence v1.0 document supplied with this file, or
+# available at https://www.pycom.io/opensource/licensing
+#
+
+# system packages
+try:
+    import uasyncio as asyncio
+except ImportError:
+    import asyncio
+
+from ..sys_imports import time, UART, Pin
```

Comment:
```suggestion
from ..sys_imports import time, Pin
```

`UART` is imported but not used I assume